apps/api/src/http/controllers/food-search.controller.ts

```diff
@@ -32,7 +32,23 @@ const ATTR_AS_RECIPE_INGREDIENT_ONLY = 2;
 
 const foodSearchController = ({
   cachedInheritableAttributesService,
-}: Pick<IoC, 'cachedInheritableAttributesService'>) => {
+  subscriberConfig,
+  logger,
+}: Pick<IoC, 'cachedInheritableAttributesService' | 'subscriberConfig' | 'logger'>) => {
+  const redisSubscriber = new RedisSubscriber({ subscriberConfig, logger });
```

Comment:
Do we need to create new subscriber here? Imo channel subscription could go to subscriber singleton, init function? Controller will be transient.

Reply:
True.